debian/changelog

```diff
@@ -7,7 +7,7 @@ matrix-synapse-py3 (1.2.0) stable; urgency=medium
   * Add Recommends and Depends for some libraries which you probably want.
 
   [ Synapse Packaging team ]
-  * New synapse release 1.2.0.
+  * New synapse release 1.2.0. 
```

Comment:
This is a very dumb change to appease the newsfile linter. Not sure of any other way other than changing the ci rules.